### ISSUE TITLE
Problem with choice validation validator

### ISSUE THREAD

**Issue by abritopach** [rxweb/rxweb#184]
## Describe the problem statement?

Hello! Is it possible to have the next case?

 **@propArray(Interest)
 @choice({ minLength: 3 })**
 interests: Array<Interest>;

Validation doesn't work for in this case

## Package version
@rxweb/

**Comment by ajayojha** [comment]
@abritopach The ```interests``` property is an Array of ```Interest``` model or it's an array of other data types(number, string, etc)?

Basically, the choice validation decorator won't work on nested FormArray, it only works on FormControl. As per the provided code you have used propArray decorator so I am assuming that is nested array object of Interest model and that will be transformed into the nested FormArray.

Please share a little bit more information, so that I can give you the exact solution.

**Comment by abritopach** [comment]
Thank you for responding @ajayojha 

The interests property is an array of Interest model

import { unique, required } from '@rxweb/reactive-form-validators';

```
export class Interest {
  @unique()
  name: string;

  @required()
  selected: boolean;
}
```

```
export class User {
  
  @required()
  username: string;

 ...

  @choice({ minLength: 3 })
  @propArray(Interest)
  interests: Array<Interest>;
}

```

The complete code is in the next repo: https://github.com/abritopach/ionic-rxweb-reactive-form-validators

It is an ionic project with angular 8.

Thanks!!

**Comment by ajayojha** [comment]
@abritopach Thanks for sharing the detailed information. Currently, the framework is not supporting the choice validation decorator on Nested FormArray.

I will draft case and try to include in next release (1.9.2) :)

**Comment by ajayojha** [comment]
@abritopach I have released the new version (1.9.3) and the feature is available in the new release. Now you can use as per below code:

```js
export class User {

    @propArray(UserAddress, { allowMaxIndex: 2 })
    addresses: UserAddress[];
}
```
Please let me know if there is any issue.

**Comment by ajayojha** [comment]
@arnaldorigo this is not available, Please create a feature request, so that I will add this feature in the upcoming release.
